js-constants.js

```diff
@@ -70,7 +71,6 @@ const JS_CONSTANTS = {
     "xlek.com",
     "yellowbook.com",
     "zabasearch.com",
-    "peoplebyname.com",
   ],
```

Comment:
Alternatively, this is just an array of simple strings, so we can throw a `.sort()` at the end of this and not have to worry about the original sort order.